packages/reactivity/__tests__/ref.spec.ts

```diff
@@ -241,7 +241,6 @@ describe('reactivity/ref', () => {
     expect(isRef(computed(() => 1))).toBe(true)
 
     expect(isRef(0)).toBe(false)
-    expect(isRef(1)).toBe(false)
```

Comment:
I don't think this should be removed. Testing `isRef` with a truthy primitive value seems like a valid test case.